### PR TITLE
adding vignesh

### PR DESCRIPTION
**Pull request by ckvignesh**
<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/652"></a>

### CODE REVIEW

File: public/data/ckvignesh.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/vigneshCodes",
+      "icon": "twitter"
+    },
+    {
+        "name": "My Open Source Contributions",
+        "url": "https://ckvignesh.notion.site/List-of-Open-Source-Contributions-09b2368ec25742a0b9f0421a0d1c9f0c",
+        "icon": "github"
```
Comment (schmelto) [inline public/data/ckvignesh.json:19]: Github logo here?

File: public/data/ckvignesh.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      "title": "Started #100DaysOfCode",
+      "image": "https://github.com/ckvignesh.png",
+      "date": "25 October 2021",
+      "icon": "github",
+      "color": "red",
+      "description": "Started this journey to upskill myself and document my learning",
+      "url": "https://twitter.com/vigneshCodes"
+    }
+  ]
+}
```
Comment (krishguptadev) [inline public/data/ckvignesh.json:41]:
```suggestion
}

```

**Review by krishguptadev — CHANGES_REQUESTED** [review]

File: public/data/ckvignesh.json (hunk 1 above)
Comment (ckvignesh) [reply public/data/ckvignesh.json:19]: Sure, I'll change it. Do you have any suggestions for it from the existing icons? Thanks,

File: public/data/ckvignesh.json (hunk 2 above)
Comment (ckvignesh) [reply public/data/ckvignesh.json:41]: Will do.

Comment (ckvignesh) [reply public/data/ckvignesh.json:41]: Isn't this the closing [] of milestones?
GT

File: public/data/ckvignesh.json (hunk 1 above)
Comment (ckvignesh) [reply public/data/ckvignesh.json:19]: I'll make it a link icon.

File: public/data/ckvignesh.json (hunk 2 above)
Comment (ckvignesh) [reply public/data/ckvignesh.json]: Thanks, @krishguptadev

File: public/data/ckvignesh.json (hunk 1 above)
Comment (ckvignesh) [reply public/data/ckvignesh.json]: Thanks, @schmelto

File: public/data/ckvignesh.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "C K Vignesh",
+  "bio": "Programmer",
```
Comment (Panquesito7) [inline public/data/ckvignesh.json:3]: Maybe add a bit more of yourself, what do you do, your passions, etc.?

Comment (ckvignesh) [reply public/data/ckvignesh.json:3]: Sure, will do it.

Comment (Vyvy-vi) [reply public/data/ckvignesh.json:3]: I mean that isn't _necessary_, so this is good to go :)

**Review by Vyvy-vi — APPROVED** [review]
lgtm

File: public/data/ckvignesh.json (hunk 3 above)
Comment (ckvignesh) [reply public/data/ckvignesh.json:3]: Thank you, @Vyvy-vi

**Pull request merged**